### PR TITLE
OKTA-486319 - Team Scotty - Added a note to previous rate limits page

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a note clarifying high rate limit capacity to: [https://developer.okta.com/docs/reference/rl-previous/#high-capacity-rate-limits](https://developer.okta.com/docs/reference/rl-previous/#high-capacity-rate-limits)
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-486319](https://oktainc.atlassian.net/browse/OKTA-486319)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-previous/index.md [hunk 1]
```diff
@@ -32,4 +32,6 @@ The following are the high capacity rate limits per minute that apply across the
 | `/bc/image/fileStoreRecord`                                                | 3000      | 3000         |
 | `/bc/globalFileStoreRecord`                                                | 3000      | 3000         |
 
+>**Note:** The increased rate limit applies only to the exact endpoints in the previous table. For example, the high-capacity rate limit only applies to `api/v1/apps/${id}` and not to paths such as `api/v1/apps/${id}/users`.
```
Comment (ShanPonnusamy-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-previous/index.md:35]: Not all of the endpoints listed above are exact match.   The following endpoints listed in the above table are not exact matches:

- /api/v1
- /api/v1/authn
- /bc/image/fileStoreRecord
- /bc/globalFileStoreRecord
- /api/v1/sessions
- /oauth2/{authorizationServerId}/v1
- /oauth2/v1

**Review by ShanPonnusamy-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/reference/rl-previous/index.md [hunk 2]
```diff
@@ -13,22 +13,22 @@ This page covers the rate limits for orgs that were created before 2018-05-17.
...
 | -------------------------------------------------------------------------- | ---------: | ------------: |
 | `/oauth2/${authorizationServerId}/v1`                                       | 6000      | 6000         |
 | `/oauth2/v1` except `/oauth2/v1/clients`                                   | 6000      | 6000         |
 | `/api/v1`                                                                  | 3000      | 3000         |
 | `/api/v1/sessions`                                                         | 3000      | 3000         |
-| `/app/template_saml_2_0/${key}/sso/saml`                                    | 3000      | 3000         |
-| `/app/${app}/${key}/sso/saml`                                                | 3000      | 3000         |
-| `/api/v1/groups/${id}`                                                      | 3000      | 3000         |
-| `/api/v1/users/${id}`                                                       | 3000      | 3000         |
+|  Only `/app/template_saml_2_0/${key}/sso/saml`                                    | 3000      | 3000         |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-previous/index.md:24]: Maybe use "Exactly" or "Specifically"?

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-previous/index.md:24]: As per slack convo, will stick with "Only" for now.

**Pull request merged**